File: src/main/java/io/github/blueminecraftteam/healthmod/common/effects/ModEffect.java
package io.github.blueminecraftteam.healthmod.common.effects;

import net.minecraft.potion.Effect;
import net.minecraft.potion.EffectType;

/**
 * Class to bypass a protected constructor in the {@link Effect effect class}.
 */
public class ModEffect extends Effect {
    public ModEffect(EffectType typeIn, int liquidColorIn) {
        super(typeIn, liquidColorIn);
    }
}

File: src/main/kotlin/io/github/blueminecraftteam/healthmod/common/effects/ModEffect.kt
/*
 * Copyright (c) 2020 Blue Minecraft Team.
 *
 * This file is part of HealthMod.
 *
 * HealthMod is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * HealthMod is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with HealthMod.  If not, see <https://www.gnu.org/licenses/>.
 */

package io.github.blueminecraftteam.healthmod.common.effects

import net.minecraft.potion.Effect
import net.minecraft.potion.EffectType

/**
 * Class to bypass a protected constructor in the [Effect class][Effect].
 */
class ModEffect(typeIn: EffectType, liquidColorIn: Int) : Effect(typeIn, liquidColorIn)